ereuse_devicehub/resources/action/schemas.py

```diff
@@ -369,15 +412,10 @@ class Prepare(ActionWithMultipleDevices):
 
 class Live(ActionWithOneDevice):
     __doc__ = m.Live.__doc__
-    ip = IP(dump_only=True)
-    subdivision_confidence = Integer(dump_only=True, data_key='subdivisionConfidence')
-    subdivision = EnumField(Subdivision, dump_only=True)
-    country = EnumField(Country, dump_only=True)
-    city = SanitizedStr(lower=True, dump_only=True)
-    city_confidence = Integer(dump_only=True, data_key='cityConfidence')
-    isp = SanitizedStr(lower=True, dump_only=True)
-    organization = SanitizedStr(lower=True, dump_only=True)
-    organization_type = SanitizedStr(lower=True, dump_only=True, data_key='organizationType')
+    final_user_code = SanitizedStr(data_key="finalUserCode", dump_only=True)
+    serial_number = SanitizedStr(data_key="serialNumber", dump_only=True)
+    time = Integer(dump_only=False)
+    hours_of_use = Integer(dump_only=False)
```

Comment:
The variable 'hours_of_use' does not appear in the Action models. To confirm that we see it the same way, time is the TestDataStorage.power_cycle_count of Snapshot Live, instead the 'hours_of_use' would be the difference between time - the time of the previous Live action or Snapshot with TestDataStorage. 
But if there is no previous data for TestDataStorage, then time == hours_of_use.

Reply:
Actualy the hours_of_use is usage_time_allocate and is a concept than we calculate and not save in the database.
But I want show in a json when arrive a query of this action.